USAGE_GUIDE.md

```diff
@@ -35,11 +74,11 @@
 
    If you deploy upcoming versions of your plugin, you must increment the build or release number accordingly (see [our docs on versioning your plugin](https://devs.aragon.org/docs/osx/how-to-guides/plugin-development/publication/versioning)).
 
-### Testing
+### Contract Testing
```

Comment:
Please create a `Contract` heading as it was done before and do not repeat the term `Contract` in every sub-heading.

Reply:
addressed in [6e1e9e1](https://github.com/aragon/osx-plugin-template-hardhat/pull/25/commits/6e1e9e1354f20ae7c695e0ea31dd55841e73ca5f)